client/src/components/HistoryExport/ExportLink.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <span>
+        <b
+            ><a :href="link">{{ link }}</a></b
+        >
```

Comment:
Minor issue, but the tag formatting seems inconsistent?

Reply:
I don't do formatting, this is all prettier's doing I think.